src/main/g8/src/main/scala/$package$/HelloWorld.scala

```diff
@@ -0,0 +1,6 @@
+package $package$
+
+object HelloWorld {
+
+  def say(): String = "Hello Cats!"
```

Comment:
I think it'll show the point more if this returns the string in IO :)

Reply:
Yes, that makes sense - I've updated my PR accordingly